LICENSE.md

```diff
@@ -1,21 +0,0 @@
-MIT License
-
-Copyright (c) 2017 Airbnb
```

Comment:
Should we just put Coveo instead of Airbnb like in other repos?

Reply:
I don't know if we can actually...i'm not so familiar with the intricacies of licensing